### ISSUE TITLE
UI suggestion

### ISSUE THREAD

**Issue by ilanit1997** [allegroai/clearml#511]
Hi, I am using clearml to track my experiments and I noticed that there are a lot of tags automatically created for different optimizations task which make my tag list long and messy ... 
Is it possible to add an option to delete tags from tag list ? 
![image](https://user-images.githubusercontent.com/80041689/145191505-17027873-e91e-4d39-a663-09c527fdd1ec.png)

**Comment by jkhenning** [comment]
Hi @ilanit1997,

As far as I know this suggestion list is taken from the tags currently used in the project - do you see a different behavior?

**Comment by ilanit1997** [comment]
I am sorry but I do not understand the question..

**Comment by ainoam** [comment]
@ilanit1997 The folder-filter button (at the top of the image you attached, next to the looking-glass icon)controls whether the list contains tags from the specific project you're in, or all of your currently used tags. Assuming a specific project makes use of a limited set of tags - this will provide what you're looking for.

WDYT?

**Comment by ilanit1997** [comment]
Yes I understand the reasoning but unfortunately we have one main project which contains our training tasks, thus the tag list has accumulated about 30 tags from different team members. In addition every mistake made in typing/small differences, cannot be reversed.

For example we have two tags : "for hpo", "for_HPO" that are basically the same in their context and are treated the same, but we cannot get rid of one of them.

**Comment by ainoam** [comment]
@ilanit1997 This last comment confuses me a little bit - Are you looking to control the tag suggestions when adding a tag, or to "remove an unnecessary tag from the system"?

For the former, notice that the list is more of an autocomplete helper, so just typing part of the tag you need will compact the contents significantly.
For the latter, the tag list simply displays all tags available in the scope (global or project) so if you have erroneous tags attached to a task - fixing those (modifying the task 'tags' field) will eliminate them from the list once no other task is making use of them.

Or is it something else you're trying to accomplish?
